test/suites/fips/validate-fips.robot

```diff
@@ -38,8 +41,15 @@ Teardown
 
 Microshift Binary Should Dynamically Link FIPS Ossl Module
     [Documentation]    Check if Microshift binary is FIPS compliant.
+
+    ${stdout_file}=/tmp/microshift-fips.run.log
+    ${stdout}    ${rc}=    Execute Command
+    ...    LD_DEBUG=symbols microshift run 2>${stdout_file}
```

Comment:
The file name seems to be "wrong". Did you mean `strerr_file`?

Reply:
Actually  it contains both stdout and stderr.